.github/labeler.yml

```diff
@@ -13,6 +13,9 @@ documentation:
       - "**/*.md"
 ci/cd:
   - changed-files:
-    - any-glob-to-any-file:
-      - ".github/**"
+    - any-glob-to-all-files:
+      - ".github/workflows/**"
+      - ".github/actions/**"
+      - ".github/dependabot.yaml"
+      - ".github/**/*.sh"
       - ".circleci/**"
```

Comment:
Is this correct? The `ci/cd` label will not be added to PRs that contain changes to e.g. workflows and actions at the same time.